packages/openactive-broker-microservice/app.js

```diff
@@ -100,7 +100,7 @@ state.incompleteFeeds.markFeedHarvestStarted('DatasetSite');
 const server = http.createServer(app);
 server.on('error', onHttpServerError);
 
-app.listen(PORT, () => {
+app.listen(PORT, async () => {
```

Comment:
I disentangled this PR from ongoing work on moving the cache into the database. Unfortunately it would be a lot of effort to revert moving all cache actions to async. Hopefully this is okay to include as it doesn't functionally change anything 🙏